src/queue.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+function Node( data ) {
+  this._data = data
+  this._next = null
+}
+
+export default class Queue {
+  constructor() {
+    this._front = null
```

Comment:
Like the use of `_` to distinguish between instance variables and local variables.